ScreenBoom/ScreenBoom/ViewModel/EventDetailViewModel.swift

```diff
@@ -12,28 +12,67 @@ import FirebaseDatabase
 
 class EventDetailViewModel: NSObject {
   
-//  var firebaseDatabaseReference2: DatabaseReference = Database.database().reference()
+  //  var firebaseDatabaseReference2: DatabaseReference = Database.database().reference()
   var eventDetail: EventDetail?
-//  let eventViewModel = EventViewModel()
+  //  let eventViewModel = EventViewModel()
   let firebaseNodeNames = FirebaseNodeNames()
   
   // Check if event detail is exist and return the eventDetail if success or error string if failure
-  func checkIfEventDetailExist (event: Event, completion: (@escaping(Result<EventDetail>) -> Void) ) {
-  Database.database().reference().child("EventDetails").child(event.eventName).observeSingleEvent(of: .value, with: {  (eventDetailSnapshot) in
+  func checkIfEventDetailExist (event: Event, completion: (@escaping(Result<EventDetail?>) -> Void) ) {
+    Database.database().reference().child("EventDetails").child(event.eventName).observeSingleEvent(of: .value, with: {  (eventDetailSnapshot) in
```

Comment:
Ive already reviewed this view controller in the other PR so I wont talk too much about it unless theres something different. Just make sure you address the comments from the other PR